pom.xml

```diff
@@ -1945,7 +1944,7 @@
       <dependency>
         <groupId>org.apache.thrift</groupId>
         <artifactId>libfb303</artifactId>
-        <version>${libthrift.version}</version>
+        <version>0.9.3</version>
```

Comment:
it's ok this is 0.9.3?

Reply:
I mean mixin with libthrift 0.12

Reply:
Yeah, there's no newer version of libfb303. It seems to work though.